app/views/spree/admin/orders/_invoice_table4.html.haml

```diff
@@ -0,0 +1,62 @@
+%table.order-summary{:width => "100%"}
+  %thead
+    %tr
+      %th{:align => "left"}
+        %h5= t(:invoice_column_item)
+      %th{:align => "right", :width => "15%"}
+        %h5= t(:invoice_column_qty)
+      %th{:align => "right", :width => "15%"}
+        %h5= @invoice_presenter.has_taxes_included ? t(:invoice_column_unit_price_with_taxes) : t(:invoice_column_unit_price_without_taxes)
+      %th{:align => "right", :width => "15%"}
+        %h5= @invoice_presenter.has_taxes_included ? t(:invoice_column_price_with_taxes) : t(:invoice_column_price_without_taxes)
+      - if @invoice_presenter.total_tax > 0
+        %th{:align => "right", :width => "15%"}
+          %h5= t(:invoice_column_tax_rate)
+  %tbody
+    - @order.sorted_line_items.each do |item|
```

Comment:
There's a few places in this `invoice_table4` file that are using `@order` instead of `@invoice_presenter`. I think they all need to be `@invoice_presenter`, otherwise the data from the invoice isn't being used.